developer/contribution.rst

```diff
@@ -39,3 +39,14 @@ New packages / extensions
 Using Python entry points it is easy to contribute extensions in separate packages.
 To ease discoverability and ensure long term maintenance if individual maintainers move on it is encouraged to host the code in a repository under the `colcon` organization unit on GitHub.
 Please open a ticket to either ask for the creation of a repository which you will have `admin` level access to or for moving an existing repository to this organization unit.
+
+Use keyword in metadata
```

Comment:
As this applies to any colcon extension and not only the "colcon metadata" extension, it may be worth disambiguate here. But maybe it's obvious in the context of this document ?

Reply:
You did you get the impression this is related to "colcon metadata" extensions? Please suggest any kind of rephrasing you have in mind.